themes/vocabulary_theme/templates/blocks/recent-posts.html

```diff
@@ -1,20 +1,24 @@
 {% from "macros/posts.html" import render_post_summary %}
 
-<section class="recent-posts">
+<section class="recent-posts" aria-labelledby="recent-posts-title">
   <div class="container">
     <div class="level">
-      <h2 class="is-paddingless level-left">
+      <h2 id="recent-posts-title" class="is-paddingless level-left">
         {{ this.title }}
       </h2>
       <span class="level-right">
-        <a class="posts-link" href="/blog/entries">See all posts <i class="icon angle-right"></i></a>
+        <a class="posts-link" href="/blog/entries" aria-label="See all blog posts">
```

Comment:
if the contents of an `anchor` already contain descriptive content thent he `aria-label` isn't necessary.